charts/kminion/values.yaml

```diff
@@ -89,6 +89,7 @@ tolerations: []
 affinity: {}
 
 customLabels: {}
+annotations: {}
```

Comment:
Given that we also have service annotations and these are deployment exclusive, I think this variable should go under `deployment`

Reply:
good catch @weeco - updated